src/main/events/misc/open-external-url.ts

```diff
@@ -0,0 +1,19 @@
+import { BrowserWindow, app, shell } from "electron";
+import { registerEvent } from "../register-event";
+
+let mainWindow: BrowserWindow | null = null;
+
+app.on("ready", () => {
+  mainWindow = new BrowserWindow({});
```

Comment:
You're creating a whole new `BrowserWindow` instance without the necessity of doing so, please remove this.